visualizer/filters-bar.js

```diff
@@ -36,51 +38,65 @@ class FiltersContainer extends HtmlContent {
       this.draw()
     })
 
+    this.ui.on('setData', () => {
+      this.draw()
+    })
     this.toggleSideBar = contentProperties.toggleSideBar.bind(this.ui)
   }
 
   initializeElements () {
     super.initializeElements()
 
     // App checkbox
-    this.d3AppCheckBox = this.d3Center.d3Element.append(() =>
-      checkbox({
-        leftLabel: 'app',
-        classNames: ['key-app'],
-        onChange: e => this.setCodeAreaVisibility('app', e.target.checked)
-      }))
+    this.d3AppCheckBox = this.d3Center.d3Element.append('div')
+      .classed('filter-option', true)
+      .classed('key-app', true)
+      .append('div')
+      .classed('label-wrapper', true)
+      .append(() =>
+        checkbox({
+          leftLabel: 'app',
+          onChange: e => this.setCodeAreaVisibility('app', e.target)
+        })
+      )
 
     // Dependencies combo ****
-    this.d3DepsCombo = this.d3Center.d3Element.append(() => dropdown({
-      classNames: ['key-deps'],
+    this.DepsDropDown = dropdown({
+      classNames: ['filter-option', 'key-deps'],
       label: checkbox({
         leftLabel: `<span class='after-bp-1'>Dependencies</span>
           <span class='before-bp-1'>Deps</span>`,
-        onChange: e => this.setCodeAreaVisibility('deps', e.target.checked)
+        onChange: e => this.setCodeAreaVisibility('deps', e.target)
       }),
-      content: 'No children... for now',
+      content: getChildrenHtml.bind(this, 'deps'),
       expandAbove: true
-    }))
+    })
+    this.d3DepsCombo = this.d3Center.d3Element.append(() => this.DepsDropDown)
```

Comment:
camel case
```suggestion
    this.d3DepsCombo = this.d3Center.d3Element.append(() => this.depsDropDown)
```

Reply:
There are a few more so it's probably easiest to address manually in one go instead of using N suggestions :)